bindings/go/src/fdb/errors_test.go

```diff
@@ -0,0 +1,62 @@
+/*
+ * errors_test.go
```

Comment:
I'm not a CMake expert, but shouldn't we add this new file to https://github.com/apple/foundationdb/blob/main/bindings/go/CMakeLists.txt#L2-L23?

Reply:
My bad, will do!